src/Relation/Binary/Bundles.agda

```diff
@@ -297,3 +297,14 @@ record StrictTotalOrder c ℓ₁ ℓ₂ : Set (suc (c ⊔ ℓ₁ ⊔ ℓ₂)) wh
   "Warning: decSetoid was deprecated in v1.3.
   Please use Eq.decSetoid instead."
   #-}
+
+
+
+record ApartnessRelation c ℓ₁ ℓ₂ : Set (suc (c ⊔ ℓ₁ ⊔ ℓ₂)) where
+  field
+    Carrier     : Set c
+    _≈_         : Rel Carrier ℓ₁
```

Comment:
We're missing fixity definitions for these relations (see other definitions above). I wonder what fixity would be best for `_#_`? I'm guessing as it's the counterpart for equality it should also be 4?

Reply:
Right I completely forgot fixities throughout. I've set `_#_` to `infix 4` as you suggested. I think that does make the most sense.